content/en/continuous_integration/pipelines/azure.md

```diff
@@ -37,50 +37,48 @@ Set up tracing on Azure Pipelines to gain real time insights into your CI/CD wor
 
 ## Configure the Datadog integration
 
-The Datadog integration for [Azure Pipelines][1] works by using [service hooks][2] to send data to Datadog.
+The Datadog integration for [Azure Pipelines][16] works by using [service hooks][2] to send data to Datadog.
 
 1. Install the [Datadog CI Visibility][8] extension from the Azure Marketplace. There are several extensions starting with **Datadog**, make sure that you are installing the [Datadog CI Visibility][8] extension.
 
 2. For each project, go to **Project settings > Service hooks** in Azure DevOps and select the green plus (+) icon to create a subscription.
 
-3. Create a new subscription to the `Datadog CI Visibility` service for each of the following webhook types:
+3. Create a subscription to the `Datadog CI Visibility` service for each of the following webhook types. These event types are required and must be enabled individually. 
+
     - **Run state changed**
     - **Run stage state changed**
     - **Run job state changed**
-    - **Run stage approval completed**
-    - **Run stage waiting for approval**
```

Comment:
Why are these two types removed?

Reply:
(Feel free to correct this if it's incorrect) but I was looking at the Azure setup in Org2 and Demo and didn't see these two:

![Screenshot 2024-08-22 at 8 36 11 AM](https://github.com/user-attachments/assets/f9ecf90c-a7ab-4a6c-9d32-303bf2bb1c4a)

Reply:
It is actually the opposite, we miss them in-app. I will ask to add them